src/com/cje/RequestPage.java

```diff
@@ -1,17 +1,633 @@
+import javax.swing.text.MaskFormatter;
+import java.awt.*;
+import java.awt.datatransfer.Clipboard;
+import java.awt.datatransfer.StringSelection;
+import java.awt.event.*;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+import java.text.ParseException;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.logging.Level;
 
 /**
- * Created by bbybby on 6/8/2016.
+ * Created by Jieun Choi on 6/8/2016.
+ *
+ * class for User interface
  */
-public class RequestPage {
+public class RequestPage extends JFrame{
     private JTextField titleTextField;
 
-    private JPanel panel1;
+    private JPanel mainPanel;
     private JPanel inputPanel;
     private JRadioButton maleRdBtn;
     private JRadioButton femaleRdBtn;
     private JComboBox birthYearCBox;
-    private JComboBox birthMonthCBox;
+    private JFormattedTextField heightTxt;
+    private JFormattedTextField weightTxt;
+    private JFormattedTextField headTxt;
+    private JFormattedTextField bmiTxt;
+    private JButton viewBtn;
+    private JFormattedTextField birthWeightTxt;
+    private JFormattedTextField birthHeightTxt;
+    private JLabel ageTxt;
+    private JLabel baseDateTxt;
+    private JFormattedTextField birthDateTxt;
+    private JButton resetBtn;
+    private JButton exitBtn;
+    private JPanel resultPanel;
+    private JLabel resultTxt;
+    private JCheckBox calculateByDayCkBox;
+    private JButton changeDateBtn;
+    private JLabel ageLabel;
+
+    private DataManager dm;
+    private ResultView rv;
+    private Params.Gender gender = Params.Gender.NONE;
+    private int birthYear;
+    private int birthMonth;
+    private int birthDay;
+    private LocalDate baseDate;
+
+    private float birthWeight;
+    private float birthHeight;
+    private float height;
+    private float weight;
+    private float head;
+    private float bmi;
+
+    private int ageByDay;
+
+    boolean isDataLoaded;
+    boolean isCalculatedByDay;
+
+    // Initialize member variables
+    private void initParam() {
+        birthYear = -1;
+        birthMonth = -1;
+        birthDay = -1;
+        ageByDay = -1;
+
+        birthWeight = -1;
+        birthHeight = -1;
+        height = -1;
+        weight = -1;
+        head = -1;
+        bmi = -1;
+
+        setBaseDate(LocalDate.now());
+    }
+
+    private void init() {
+        initParam();
+
+        dm = new DataManager();
+        rv = new ResultView();
+
+        // Load Data
+        isDataLoaded = false;
+        isCalculatedByDay = true;
+
+        isDataLoaded = dm.loadData();
+        if(!isDataLoaded) {
+            Utils.showMessage("Data loading failed!");
+            Utils.LOGGER.log(Level.SEVERE, "File reading failed.");
+        }
+    }
+
+    public RequestPage() {
+        super("Growth Indicator");
+
+
+        setContentPane(mainPanel);
+        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        pack();
+        setLocationRelativeTo(null);
+
```

Comment:
I think all the code below is too much for constructor.
Basically, constructor should not do too much.
For example, in your main code,
```
    public static void main(String[] args) {
        RequestPage mainFrm = new RequestPage();
        mainFrm.setTitle("GrowthIndicator version 0.9 Beta");
        mainFrm.setSize(800, 720);
        mainFrm.setVisible(true);
    }
```
I think  you can have more apis so that
```
    public static void main(String[] args) {
        RequestPage mainFrm = new RequestPage();
        mainFrm.initialize();
        mainFrm.addListeners();
        mainFrm.setTitle("GrowthIndicator version 0.9 Beta");
        mainFrm.setSize(800, 720);
        mainFrm.setVisible(true);
    }
```